front/src/modules/simulationResult/components/SpaceTimeChart/withOSRDData.tsx

```diff
@@ -47,6 +48,7 @@ function withOSRDData<T extends SpaceTimeChartProps>(
     const timePosition = useSelector(getTimePosition);
     const simulation = useSelector(getPresentSimulation);
     const isPlaying = useSelector(getIsPlaying);
+    const chartXScaleDomain = useSelector(getChartXScaleDomain);
```

Comment:
I think we don't need to store this var in the store, it can simply be a state variable in SimulationResults. We should avoid adding new information in the store (unless it's absolutely necessary). Moreover Chaka and Uriel are refactoring it, so if you want to keep it in the store, you should add it to osrdConf2 I believe